contracts/src/bridge/Bridge.sol

```diff
@@ -53,6 +54,8 @@ contract Bridge is Initializable, DelegateCallAware, IBridge {
     IOwnable public rollup;
     address public sequencerInbox;
 
+    uint256 public override sequencerReportedSubMessageCount;
```

Comment:
would be good to add some docs around this
```suggestion
    /// @notice This shouldn't be trusted in the system since the sequencer has full control over it. It is used to help ensure consistency when there are multiple batches in flight.
    uint256 public override sequencerReportedSubMessageCount;
```